script.js

```diff
@@ -35,29 +35,16 @@ var list = [
 	/you won['’]?t believe/i
 ];
 
-function test( string ) {
-
+function isClickbait( string ) {
 	if ( string.length < 20 || string.length > 100 ) return false;
-
-	for ( var i = 0, l = list.length; i < l; i ++ ) {
-
-		if ( list[ i ].test( string ) ) return true;
-
-	}
-
-	return false;
-
+	return list.some( clickbait => clickbait.test( string ) );
```

Comment:
I hope I understand you correctly. I added the logging of clickbaits in a new commit.